docs/source/en/model_doc/prism.md

```diff
@@ -0,0 +1,110 @@
+
+The model was proposed in [Automatic Machine Translation Evaluation in Many Languages via Zero-Shot Paraphrasing](https://aclanthology.org/2020.emnlp-main.8.pdf) by Brian Thompson and Matt Post.
+
+The abstract from the paper is the following:
+
+*We frame the task of machine translation evaluation as one of scoring machine translation output with a sequence-to-sequence paraphraser, conditioned on a human reference. Wepropose training the paraphraser as a multilingual NMT system, treating paraphrasing as a zero-shot translation task (e.g., Czech to Czech). This results in the paraphraser’s output mode being centered around a copy of the input sequence, which represents the best case scenario where the MT system output matches a human reference. Our method is simple and intuitive, and does not require human judgements for training. Our single model (trained in 39 languages) outperforms or statistically ties with all prior metrics on the WMT 2019 segment-level shared metrics task in all languages (excluding Gujarati where the model had no training data). We also explore using our model for the task of quality estimation as a metric—conditioning on the source instead of the reference—and find that it significantly outperforms every submission to the WMT2019 shared task on quality estimation in every language pair.*
+
+This model was contributed by [dariast](https://huggingface.co/dariast/).
+The original code can be found [here](https://github.com/thompsonb/prism/tree/master) and the original documentation is found [here](https://github.com/thompsonb/prism/blob/master/translation/README.md).
+
+
+## Usage tips
+
+To use `PrismTokenizer`, ensure that the `sentencepiece` package is installed, as it is a required dependency for handling multilingual tokenization.
+
+```bash
+pip install sentencepiece
+```
+
+## Example
+```python
+from transformers import PrismForConditionalGeneration, PrismTokenizer
+
+uk_text = "Життя як коробка шоколаду"
+ja_text = "人生はチョコレートの箱のようなもの。"
+
+model = PrismForConditionalGeneration.from_pretrained("dariast/prism")
+tokenizer = PrismTokenizer.from_pretrained("dariast/prism")
+
+# Translate Ukrainian to French
+tokenizer.src_lang = "uk"
+encoded_uk = tokenizer(uk_text, return_tensors="pt")
+generated_tokens = model.generate(**encoded_uk, forced_bos_token_id=tokenizer.get_lang_id("fr"), max_new_tokens=20)
+print(tokenizer.batch_decode(generated_tokens, skip_special_tokens=True))
+# => 'La vie comme une boîte de chocolat.'
+
+# Translate Japanese to English
+tokenizer.src_lang = "ja"
+encoded_ja = tokenizer(ja_text, return_tensors="pt")
+generated_tokens = model.generate(**encoded_ja, forced_bos_token_id=tokenizer.get_lang_id("en"), max_new_tokens=20)
+print(tokenizer.batch_decode(generated_tokens, skip_special_tokens=True))
+# => 'Life is like a box of chocolate.'
+```
+
+## Languages Covered
```

Comment:
Note that the list of supported languages is specific to a model checkpoint, not to the architecture itself. So I think the language list belongs into the Model Card, but not the documentation of the architecture.

Reply:
✅Fixed